wagtail/admin/templates/wagtailadmin/shared/locale_selector.html

```diff
@@ -1,27 +1,20 @@
-{# Based on listing/_button_with_dropdown.html #}
 {% load wagtailadmin_tags %}
+
+{% fragment as locale_label %}
+    {% icon name="site" classname="middle" %}
+    {{ locale.get_display_name }}
+{% endfragment %}
+
 {% if translations %}
-    <div class="c-dropdown t-inverted {{ class }}" style="display: inline-block;" data-dropdown>
-        <a href="javascript:void(0)" aria-label="{{ locale.get_display_name }}" class="c-dropdown__button u-btn-current w-no-underline">
-            {% icon name="site" classname="default" %}
-            {{ locale.get_display_name }}
-            <div data-dropdown-toggle class="o-icon c-dropdown__toggle c-dropdown__togle--icon">
-                {% icon name="arrow-down" %}{% icon name="arrow-up" %}
-            </div>
-        </a>
-        <div class="t-dark">
-            <ul class="c-dropdown__menu u-toggle  u-arrow u-arrow--tl u-background">
-                {% for translation in translations %}
-                    <li class="c-dropdown__item ">
-                        <a href="{{ translation.url }}" aria-label="{{ translation.locale.get_display_name }}" class="u-link is-live w-no-underline">
-                            {{ translation.locale.get_display_name }}
-                        </a>
-                    </li>
-                {% endfor %}
-            </ul>
-        </div>
-    </div>
+    {% fragment as toggle_classname %}{% if theme == "large" %}button{% else %}button button-small{% endif %} {{ class }}{% endfragment %}
```

Comment:
Not my favourite bit of work but considering this "large" variant is only used in two places, and was already defined as a "large" variant, this felt more appropriate than updating every other usage to say "small variant".